Client/cypress/support/e2e.ts

```diff
@@ -0,0 +1,21 @@
+// ***********************************************************
```

Comment:
comments kan van mij ook weg

Reply:
Dit is de standaard documentatie van Cypress. Dit houd in dat de code geen hint geeft waar de documentatie wel te vinden is. Dus alle comments worden verwijderd.